r/tests/testthat/test-dataset.R

```diff
@@ -194,6 +194,38 @@ test_that("Hive partitioning", {
   )
 })
 
+test_that("open_dataset fails when partitioning is passed as schema", {
```

Comment:
I'd call this something about "input validation"

Reply:
all set ! when it passes CI, it shall be ready to merge